### ISSUE TITLE
Not starting when running in a seocnd code window

### ISSUE THREAD

**Issue by HankB** [dbankier/vscode-instant-markdown#22]
1) I opened a new VS code window (<shift><ctrl><n>) and drug README.md from the original window to the new window.
2) Close README.md in original window. (Browser view nowe closed.)
3) Editing README.md in new window - no browsaer preview. :(
4) Close and reopen the README.md file in new window - no change. (There are no Markdown files open in the original window.)
5) Some time later I open Preferences->Settings (settings.json) for an unrelated issue. When I save and switch back to the README.md, the preview appears. :)

- possible competition for server port? (I don't know if the new window runs in the same process or a new process.)
- possible need to switch between files to trigger display?

Thanks!

**Comment by dbankier** [comment]
Yep that is it the issue. The new implementation is running the server from within the plugin, so it a "port already in use" issue. I'll think of the best way around this...

**Comment by HankB** [comment]
Without knowing anything about the code (but familiar with 'port already in use') would incrementing the port number when that error occurs work?
